### DISCUSSION TITLE
Engine platforms

### DISCUSSION THREAD

**Discussion by gsvarovsky** [m-ld/m-ld-spec#96] [Discussion]
Discussion topic for people's engine platform requirements!

Please pop in a comment if you have ideas, or go straight to these specific links:
- #97 
- #54 
- #40 
- #42
- #39 
- #41 
- #95
- #100

[(Query for platform Issues)](https://github.com/m-ld/m-ld-spec/issues?q=label%3Aplatform)

**Comment by jbcpollak** [comment]
Golang platform support would be great!

**Reply by gsvarovsky** [reply]
Thanks Joshua! I've recently had reason to use Go and I'm fascinated...
